SeriesCruncher/app/src/main/java/com/balazs_csernai/seriescruncher/utils/common/DateUtils.java

```diff
@@ -0,0 +1,45 @@
+package com.balazs_csernai.seriescruncher.utils.common;
+
+import android.util.Log;
+
+import com.balazs_csernai.seriescruncher.utils.converter.EpisodeDetailsConverter;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+
+/**
+ * Created by erik_markus_kramli on 2016-02-09.
+ */
+public class DateUtils {
+
+    public static final String DATE_PATTERN = "yyyy-MM-dd";
+    private static final DateFormat DATE_FORMAT = new SimpleDateFormat(DATE_PATTERN, Locale.ENGLISH);
+
+    public static Date parseDate(String rawDate) {
+        try {
+            return DateUtils.DATE_FORMAT.parse(rawDate);
+
+        } catch (ParseException e) {
+            Log.e(DateUtils.class.getSimpleName(), "Error parsing date: " + rawDate);
+            return null;
+        }
+    }
+
+    public static String parseDate(Date date) {
+        if (date == null) {
+            return null;
+        } else {
+            return DATE_FORMAT.format(date);
+        }
+    }
+
+    public static Date getTodayDate() {
+        return Calendar.getInstance().getTime();
```

Comment:
It would be better to use current date as a singleton. Calendar.getInstance() creates a new instance every time.